### PR TITLE
Added support for valetudo

### PR DESCRIPTION
**Pull request by pkishino**
probably not the prettiest, but it works to support valetudo vacuums as per #8

### CODE REVIEW

File: src/vacuum-card.js [hunk 1]
```diff
@@ -104,6 +105,31 @@ class VacuumCard extends LitElement {
     this.callService('set_fan_speed', { fan_speed });
   }
 
+  getStatusValues() {
+    this.type =
+      typeof this.entity.attributes.valetudo_state === 'undefined'
+        ? 'r5_max'
```
Comment (denysdovhan) [inline src/vacuum-card.js:111]: I don't like we need to specify model manually. I believe this should be as generic as possible.
We can just check if, for example, `cleaned_area ` has any value, if not, then fallback to `cleanArea ` and so one.

**Comment by denysdovhan** [comment]
Similar to #11

File: src/vacuum-card.js (hunk 1 above)
Comment (pkishino) [reply src/vacuum-card.js:111]: Yeah, sure, can remove the model type..
Were you thinking to check each variable and fall back? From what I know, valetudo user the same attributes independent of model.

Comment (denysdovhan) [reply src/vacuum-card.js:111]: I've almost finished it myself. Wait for a fix.

**Comment by denysdovhan** [comment]
Should be fixed in `v0.3.3`

**Comment by denysdovhan** [comment]
Please, note breaking changes introduced by #29. It will be released as 1.0.0.

**Pull request closed without merging**